frontend/src/views/pricing-view.js

```diff
@@ -0,0 +1,151 @@
+                    margin: 0;
+                }
+                
+                @media screen and (max-width: 759px) {
+                    #header {
+                        text-align: center;
+                    }
+                    
+                    #header vaadin-menu-bar {
+                        display: inline-block;
+                        max-width: 100%;
+                    }
+                }
+                
+                @media screen and (min-width: 760px) {
+                    #header {
+                        display: flex;
+                        align-items: baseline;
+                    }
+                    
+                    #header h5 {
+                        flex: 1;
+                    }
+                }
+                
+                .card-list {
+                    display: flex;
+                    margin: var(--lumo-space-xl) calc(var(--lumo-space-m) * -1);
+                    flex-wrap: wrap;
+                }
+                
+                .card-list > ex-card {
+                    flex: 1 0 210px;
+                    margin: var(--lumo-space-m);
+                }
+                
+                ex-card ul,
+                #footer ul {
+                    padding: 0;
+                }
+                
+                ex-card li,
+                #footer li {
+                    list-style: none;
+                }
+                
+                ex-card vaadin-button {
+                    width: 100%;
+                    font-size: var(--lumo-font-size-l);
+                    --lumo-button-size: 2.4em;
+                }
+                
+                .hero {
+                    margin: 0 auto;
+                    max-width: 700px;
+                    text-align: center;
+                }
+                
+                .hero p {
+                    font-size: var(--lumo-font-size-l);
+                }
+
+                #footer {
+                    margin-top: var(--lumo-space-xl);
+                    border-top: 1px solid var(--lumo-contrast-30pct);
+                    padding-top: var(--lumo-space-xl);
+                    padding-bottom: var(--lumo-space-l);
+                    display: flex;
+                    flex-wrap: wrap;
+                    --lumo-primary-text-color: var(--lumo-contrast-70pct);
+                }
+                
+                #footer > div {
+                    flex: 1 0 20%;
+                }
+                
+                #footer h5 {
+                    font-size: var(--lumo-font-size-l);
+                }
+                
+                #footer ul {
+                    margin: var(--lumo-space-s) 0;
+                }
+                
+                .copyright {
+                    font-size: var(--lumo-font-size-s);
+                    color: var(--lumo-contrast-60pct);
+                }
+                
+                [icon^="vaadin:vaadin-"] {
```

Comment:
IMO this could be improved by using `.copyright-box > [icon]`. 
Not a blocker though.

Reply:
My intention was to make it generic so all Vaadin logo icons would be colored with the correct brand color for the logo and if there was some other icons, those wouldn't be affected.

But since there is in fact only one icon here, I guess a simpler (easier to understand) selector would be better for examples sake.

I'll update it to use your suggestion.

Reply:
Updated.